components/ruby/lib/chef-licensing/config.rb

```diff
@@ -47,6 +47,10 @@ def license_add_command
       def license_list_command
         @license_list_command ||= "license list"
       end
+
+      def make_licensing_optional
```

Comment:
do you have a strong opinion about this name `make_licensing_optional`? can this be simply `enable_checks?` because the config is already in context of licensing.

Reply:
This was mentioned on JIRA ticket, so I only followed that. Since that will be a global flag set from the external gem or elsewhere, it should give a clear meaning. `enable_checks` alone wont tell us that it is for enabling licensing right?

Reply:
yes still within the context of `ChefLicensing`. I feel it is redundant honestly.